Device.md

```diff
@@ -407,6 +408,60 @@ gantt
 ```
 When the value of this register is above 0 (zero), the device’s timestamp will be offset by this amount. The register is sensitive to 500 microsecond increments. This register is non-volatile.
 
+
+#### **`R_HEARTBEAT` (U16) – Heartbeat register reporting the current status of the device**
+
+Address: `018`
```

Comment:
Should we fix this address now? Or do we need to wait for merging https://github.com/harp-tech/protocol/pull/39 and https://github.com/harp-tech/protocol/pull/38 beforehand?

In general I guess this is an interesting question regarding numbering of newly proposed common registers, should the address be part of the proposal, or should we number them incrementally as they get merged on a first-come, first-served basis?

Reply:
I dont think there is any requirement for registers to be sequential right now. Or at least in the protocol. I think that in general we should honor the "reserved" register of the PR in case people implemented a proof-of-concept already. But maybe there is a stronger reason to follow the sequence?

Reply:
The mentioned PRs have been merged so this is a non-issue now.